### PR TITLE
Update showcase.json

### PR DESCRIPTION
**Pull request by anunomac**
Pull for Final Demo Checklist

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @anunomac,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/405) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

**Comment by jennifertrin** [comment]
> Pull for Final Demo Checklist

@anunomac Can you please go through the CLA steps?

File: showcase.json [hunk 1]
```diff
@@ -3057,6 +3057,28 @@
     "youtube": "https://www.youtube.com/@usher_web3",
     "submittableId": "36108625"
   },
+  {
+  "name": "TIKITWORLD - kybra-tikit",
```
Comment (jennifertrin) [inline showcase.json:3061]: The goal of this Ecosystem page is to showcase the product more than the core technology. Can you please update the name to "TIKITWORLD" only?

File: showcase.json [hunk 2]
```diff
@@ -3057,6 +3057,28 @@
...
+  "description": "Kybra-Tikit manages the minting and transfers of non-fungible tikits which represent tickets to real-world events. It provides a decentralized framework for event ticketing while still allowing for managed wallets and centralized payment processing.",
```
Comment (jennifertrin) [inline showcase.json:3062]: Can you please update the description to talk about the TIkitworld as a product? I like the description on the website.

File: showcase.json [hunk 3]
```diff
@@ -3057,6 +3057,28 @@
...
+  "website": "https://tikitworld.com/",
+  "logo": "/img/showcase/tikitworld-kybra-tikit_logo.webp",
+  "screenshots": [
+    "/img/showcase/tikitworld-kybra-tikit_screenshot.webp"
+  ],
+  "id": "tikitworld-kybra-tikit",
+  "oneLiner": "Ticketing Smart contract in Kybra for deployment as an IC canister",
+  "stats": "In development",
+  "tags": [
+    "Tools / Infrastructure",
```
Comment (jennifertrin) [inline showcase.json:3072]: Can you please remove "Tools / Infrastructure" as a tag?

File: showcase.json [hunk 4]
```diff
@@ -3057,6 +3057,28 @@
...
+  {
+  "name": "TIKITWORLD - kybra-tikit",
+  "description": "Kybra-Tikit manages the minting and transfers of non-fungible tikits which represent tickets to real-world events. It provides a decentralized framework for event ticketing while still allowing for managed wallets and centralized payment processing.",
+  "website": "https://tikitworld.com/",
+  "logo": "/img/showcase/tikitworld-kybra-tikit_logo.webp",
+  "screenshots": [
+    "/img/showcase/tikitworld-kybra-tikit_screenshot.webp"
+  ],
+  "id": "tikitworld-kybra-tikit",
+  "oneLiner": "Ticketing Smart contract in Kybra for deployment as an IC canister",
```
Comment (jennifertrin) [inline showcase.json:3069]: Can you please update the one-liner as well to be more product-focused?

File: showcase.json [hunk 5]
```diff
@@ -3057,6 +3057,28 @@
     "youtube": "https://www.youtube.com/@usher_web3",
     "submittableId": "36108625"
   },
+  {
+  "name": "TIKITWORLD - kybra-tikit",
+  "description": "Kybra-Tikit manages the minting and transfers of non-fungible tikits which represent tickets to real-world events. It provides a decentralized framework for event ticketing while still allowing for managed wallets and centralized payment processing.",
+  "website": "https://tikitworld.com/",
+  "logo": "/img/showcase/tikitworld-kybra-tikit_logo.webp",
```
Comment (jennifertrin) [inline showcase.json:3064]: Can you please include a logo of Tikitworld with this path in the repo?

File: showcase.json [hunk 6]
```diff
@@ -3057,6 +3057,28 @@
...
+  "screenshots": [
+    "/img/showcase/tikitworld-kybra-tikit_screenshot.webp"
+  ],
+  "id": "tikitworld-kybra-tikit",
+  "oneLiner": "Ticketing Smart contract in Kybra for deployment as an IC canister",
+  "stats": "In development",
```
Comment (jennifertrin) [inline showcase.json:3070]: Can you please focus on your stats on the number of users that you have?

File: showcase.json [hunk 7]
```diff
@@ -3057,6 +3057,28 @@
     "youtube": "https://www.youtube.com/@usher_web3",
     "submittableId": "36108625"
   },
+  {
+  "name": "TIKITWORLD - kybra-tikit",
+  "description": "Kybra-Tikit manages the minting and transfers of non-fungible tikits which represent tickets to real-world events. It provides a decentralized framework for event ticketing while still allowing for managed wallets and centralized payment processing.",
+  "website": "https://tikitworld.com/",
+  "logo": "/img/showcase/tikitworld-kybra-tikit_logo.webp",
+  "screenshots": [
```
Comment (jennifertrin) [inline showcase.json:3065]: Can you please include a screenshot with this path in the repo?

**Review by jennifertrin — CHANGES_REQUESTED** [review]

**Comment by jennifertrin** [comment]
@anunomac Can you please also make the updates suggested in the conversation above? Please note that the Ecosystem page is meant to promote the product.

**Comment by jennifertrin** [comment]
> @anunomac Can you please also make the updates suggested in the conversation above? Please note that the Ecosystem page is meant to promote the product.
> 
> Thank you for going through the CLA steps.

@anunomac Can you please review and update by Wednesday, January 3rd?

**Comment by anunomac** [comment]
Hey jenni, apologies for the delay.
Everything you requested is done, except the stats. What stats would you like us to include and in what form?

Thank you.

**Comment by jennifertrin** [comment]
> Hey @jennifertrin , apologies for the delay. Everything you requested is done, except the stats. What stats would you like us to include and in what form?
> 
> Thank you.

Thank you. Please put down number of users in the format of "x users"

File: showcase.json (hunk 6 above)
Comment (anunomac) [reply showcase.json]: hi @jennifertrin i have updated the stats

**Comment by jennifertrin** [comment]
> Hey @jennifertrin , apologies for the delay. Everything you requested is done, except the stats. What stats would you like us to include and in what form?
> 
> Thank you.

Hello @anunomac, are you sure that you added the logo and screenshot? I do not see them included in the commit.

**Comment by jennifertrin** [comment]
@anunomac Can you please review the comments above?

**Comment by jennifertrin** [comment]
> @anunomac Can you please review the comments above?

@anunomac Are you sure that you added the logo and screenshot? I do not see them included in the commit.

**Comment by jennifertrin** [comment]
@anunomac Due to lack of response, we will close this PR.

**Pull request closed without merging**